rgw/v2/tests/s3_swift/test_sse_s3_kms_with_vault.py

```diff
@@ -221,6 +221,11 @@ def test_exec(config, ssh_con):
                                 rgw_conn,
                                 each_user,
                             )
+                    if config.test_sync_consistency_bucket_stats:
+                        log.info("Verify the bugs 2236643 and 2247742")
```

Comment:
@viduship , could add the BZs are comment & add the verify for consistency in log.info

Reply:
done.